src/pages/matching.tsx

```diff
@@ -138,6 +143,25 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props:
         .map(atco => naptanInfo.find(s => s.atcoCode === atco))
         .filter((stop: Stop | undefined): stop is Stop => stop !== undefined);
 
+    if (!matchingCookie) {
+        return {
+            props: {
+                stops: orderedStops,
+                userFareStages,
+                service: {
+                    lineName,
+                    nocCode,
+                    operatorShortName: service.operatorShortName,
+                },
+                error: false,
+            },
+        };
+    }
+
+    const matchingObject = JSON.parse(matchingCookie);
+
+    const { error } = matchingObject;
+
```

Comment:
If the below return contains `error: !matchingCookie ? false : JSON.parse(matchingCookie).error,` this should do the same job as the above  😄

Reply:
Made the change as you suggested👍